roles/icinga_downtime/tasks/add_downtimes.yml

```diff
@@ -1,82 +1,65 @@
 ---
-- name: "Add icinga host downtimes"
+- name: Add icinga host downtimes
   ansible.builtin.uri:
-    url: "{{ icinga_api_uri }}:{{ icinga_api_port }}/v1/actions/schedule-downtime?type=Host"
-    user: "{{ icinga_api_user }}"
-    password: "{{ icinga_api_password }}"
+    url: '{{ icinga_api_uri }}:{{ icinga_api_port }}/v1/actions/schedule-downtime?type=Host'
+    user: '{{ icinga_api_user }}'
+    password: '{{ icinga_api_password }}'
     validate_certs: true
     method: POST
     status_code: 200
     return_content: true
     headers:
-      Accept: 'application/json'
+      Accept: application/json
     body_format: json
-    body: '{
-             "filter": "host.name==\"{{ item }}\"",
-             "author": "{{ author }}",
-             "comment": "{{ comment }}",
-             "start_time": "{{ start_time_now }}",
-             "end_time": "{{ end_time }}",
-             "pretty": true,
-             "fixed": true,
-           }'
+    body: '{ "filter": "host.name==\"{{ item }}\"", "author": "{{ author }}", "comment":
+      "{{ comment }}", "start_time": "{{ start_time_now }}", "end_time": "{{ end_time
+      }}", "pretty": true, "fixed": true, }'
```

Comment:
this feelds a bit like our recent JSON/YAML discussion. I like the structured layout with one key per line better

same below

Reply:
Yeah, reverted.